bin/cli.js

```diff
@@ -62,13 +62,15 @@ program
   });
 
 program.on('--help', function() {
+  /* eslint-disable no-console */
```

Comment:
Since this is a CLI, should we just allow console globally in the `eslintrc`?

Reply:
Ok, I'll change that